stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/navigation/destinations/AttachmentDestination.kt

```diff
@@ -22,13 +22,16 @@ public open class AttachmentDestination(
     context: Context,
 ) : ChatDestination(context) {
 
+    private val logger = ChatLogger.get("AttachmentDestination")
+
     override fun navigate() {
         showAttachment(message, attachment)
     }
 
     public fun showAttachment(message: Message, attachment: Attachment) {
         if (attachment.type == ModelType.attach_file ||
             attachment.type == ModelType.attach_video ||
+            attachment.type == ModelType.attach_audio ||
```

Comment:
What about us? We still continue to send audio attachments as files? And therefore they are not playable on IOS?